olmo/train.py

```diff
@@ -737,6 +742,9 @@ def train_step(self, batch: Dict[str, Any], reduce_global_loss: bool = True) ->
                 self.cfg.max_grad_norm_ratio, self.scheduler_current, self.scheduler_max
             )
 
+            if group["name"] == "embedding_decay_group":
+                group["weight_decay"] *= emb_decay_factor
```

Comment:
Does't this multiply up `emb_decay_factor` across batches? It feels like this should just be set, not multiplied? Or is there some other bit that resets `group["weight_decay"]` every time?